smart-contracts/cookbook/smart-contracts/deploy-nft/nft-remix.md

```diff
@@ -24,62 +24,46 @@ This guide demonstrates how to deploy an [ERC-721](https://eips.ethereum.org/EIP
 
 Navigate to [Remix](https://remix.ethereum.org/){target=\_blank} in your web browser.
 
-The interface will load with a default workspace containing sample contracts. In this interface, you can access a file explorer, edit your code, interact with various plugins for development, and use a terminal.
+The interface will load with a default workspace containing sample contracts. In this interface, you can access a file explorer, edit your code, interact with various development plugins, and use a terminal.
 
 ## Create Your Contract
 
 1. Create a new file `contracts/MyNFT.sol`.
-2. Paste the following code:
+2. Now, paste the following ERC-721 contract code into `MyNFT.sol`:
 
     ```solidity title="contracts/MyNFT.sol"
-    // SPDX-License-Identifier: MIT
-    pragma solidity ^0.8.20;
-
-    import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
-    import "@openzeppelin/contracts/access/Ownable.sol";
-
-    contract MyNFT is ERC721, Ownable {
-        uint256 private _nextTokenId;
-
-        constructor(address initialOwner)
-            ERC721("MyToken", "MTK")
-            Ownable(initialOwner)
-        {}
-
-        function safeMint(address to) public onlyOwner {
-            uint256 tokenId = _nextTokenId++;
-            _safeMint(to, tokenId);
-        }
-    }
+    --8<-- 'code/smart-contracts/cookbook/smart-contracts/deploy-erc721-nft/nft-remix/MyNFT.sol'
     ```
 
-![](/images/smart-contracts/cookbook/smart-contracts/deploy-nft/remix/remix-01.webp)
+    ![](/images/smart-contracts/cookbook/smart-contracts/deploy-nft/nft-remix/remix-01.webp)
 
-## Compile
+## Compile the Contract
 
 1. Navigate to the **Solidity Compiler** tab (third icon in the left sidebar).
 2. Click **Compile MyNFT.sol** or press `Ctrl+S`.
 
-![](/images/smart-contracts/cookbook/smart-contracts/deploy-nft/remix/remix-02.webp)
+![](/images/smart-contracts/cookbook/smart-contracts/deploy-nft/nft-remix/remix-02.webp)
```

Comment:
In the other guides we mention the green check mark to notify users that it's been successfully compiled, let's add that here too please

Reply:
We're also missing some of the explanations we have on the ERC-20 guide. So can you please review and make sure we're being consistent? All of the sections on this page seem just very brief compared to the ERC-20 guide